go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
 	github.com/go-ole/go-ole v1.2.4 // indirect
+	github.com/jaypipes/pcidb v0.5.0
```

Comment:
As you have added the pcidb dependency you can show the amount of drivers from each vendor when termiboard is run with --all or without any flag

Reply:
Yeah sure that is possible.